lib/roku-adapter.js

```diff
@@ -33,9 +33,9 @@ class RokuAdapter extends Adapter {
    * Attempt to add any configured devices.
    */
   addKnownDevices() {
-    for (const addr of this.config.devices) {
+    for (let addr of this.config.devices) {
       if (!/^http:\/\/\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}:\d+$/.test(addr)) {
-        continue;
+        addr = `http://${addr}:8060`;
```

Comment:
I'm not sure whether roku-client has a dramatic failure if the address is something invalid, but it might be good to have an ip addr regex guarding this

Reply:
Good idea. Done.